packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -517,14 +536,13 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderBaseProps, ID
         while (startIndex < endIndex) {
           const middleIndex = Math.ceil((endIndex + startIndex!) / 2);
           if (
-            eventXRelativePosition >= this._dropHintDetails[middleIndex].startX &&
-            eventXRelativePosition <= this._dropHintDetails[middleIndex].endX
+            this._liesBetween(eventXRelativePosition, this._dropHintDetails[middleIndex].startX, this._dropHintDetails[middleIndex].endX)
           ) {
             indexToUpdate = middleIndex;
             break;
-          } else if (eventXRelativePosition < this._dropHintDetails[middleIndex].originX) {
+          } else if (this._isBefore(eventXRelativePosition, this._dropHintDetails[middleIndex].originX)) {
             endIndex = middleIndex;
-          } else if (eventXRelativePosition > this._dropHintDetails[middleIndex].originX) {

```

Comment:
It looks like the original comparison was >, but the new `_isAfter` method checks >=. Is this intentional?

Reply:
@arkogupta would you be able to clarify the above?

Reply:
I didn't want to add an extra equality check ( `|| eventXRelativePosition  === this._dropHintDetails[...]...`) in lines 508 and 510, to keep it short. So used it in the utility method and reused it here as well, where we don't actually require the equality check. But doesn't make any difference in the logic as the equality case would be filtered by the first if condition anyway. Hence kept it this way.

Reply:
Okay, I think that's fine